client/galaxy/scripts/utils/uploadbox.js

```diff
@@ -97,7 +97,7 @@
         }, false);
 
         // send request
-        Galaxy.emit.debug('uploadbox::uploadpost()', 'Posting following data.', cnf);
+        Galaxy.logger.emit('debug', 'uploadbox::uploadpost()', [ 'Posting following data.', cnf ]);
```

Comment:
So - the only reason uploadbox is different than the other changed logging statements is that it doesn't require Utils. It's only equivalent if we keep the changes you made to the global object. I moved your emit object to Utils but it's unavailable here. We can make it the same as the others if we require Utils.